src/app.js

```diff
@@ -1,21 +1,28 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+  const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+  const { searchParams } = normalizedUrl;
+
+  const parts = normalizedUrl.pathname.slice(1).split('/');
+  const query = {};
+
+  for (const key of searchParams.keys()) {
+    const value = searchParams.getAll(key);
+
+    query[key] = value.length > 1
+      ? value
+      : value[0];
+  }
```

Comment:
There is an approach to get all searchParams as an object in the module video "How to read request params"
```suggestion
  const query = Object.fromEntries(searchParams.entries())
```

Reply:
I am aware of that approach, but it doesn't work for params with multiple values

Reply:
true, my bad